recipe/meta.yaml

```
@@ -132,7 +125,8 @@ outputs:
         - ffmpeg                # [not win]
         - freetype
         - harfbuzz
-        - libglib
+        - glib                             # [not osx]
+        - libglib                          # [osx]
```

```suggestion
        - libglib
```
Is there a reason to provide the `glib` headers at runtime with OpenCV?

---

Yeah, there was a specific issue on `osx-arm64` after this commit  [26a97c9](https://github.com/AnacondaRecipes/opencv-feedstock/pull/38/commits/26a97c9cef3c660f8897ca849360fd83e3eca136):
```
ERROR (opencv,lib/libopencv_gapi.4.10.0.dylib): Needed DSO lib/libgobject-2.0.0.dylib found in ['defaults/osx-arm64::libglib==2.78.4=h0a96307_0']

  ERROR (opencv,lib/libopencv_gapi.4.10.0.dylib): .. but ['defaults/osx-arm64::libglib==2.78.4=h0a96307_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)

  ERROR (opencv,lib/libopencv_gapi.4.10.0.dylib): Needed DSO lib/libglib-2.0.0.dylib found in ['defaults/osx-arm64::libglib==2.78.4=h0a96307_0']

  ERROR (opencv,lib/libopencv_gapi.4.10.0.dylib): .. but ['defaults/osx-arm64::libglib==2.78.4=h0a96307_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```
  
  with:
`WARNING (opencv): run-exports library package defaults/osx-arm64::glib==2.78.4=h313beb8_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)`

@bwingconda  I wonder if it was caused by not providing staging channels for qt5compat, qtdeclarative, qtshadertools, and qtsvg. I'l try to address your comments and will see if it will be solved.

---

I think this is already fixed.

---

I think those were just saying you needed `libglib` as a runtime requirement and that `glib` wasn't providing anything linkable since it's mostly headers. I was just curious if the headers were needed by opencv headers when another package that uses opencv was being compiled.